src/aurelia-loader-webpack.ts

```diff
@@ -148,7 +104,7 @@ export class WebpackLoader extends Loader {
         throw new Error(`Plugin ${loaderPlugin} is not registered in the loader.`);
       }
       if (module.hot && plugin.hot) {
-        module.hot.accept(moduleId, () => plugin.hot(moduleId));
+        module.hot.accept(moduleId, () => plugin.hot!(moduleId));
```

Comment:
No, on purpose. Currently TS raises an error here because it thinks `plugin.hot` might be undefined. It knows `plugin` is not null because you checked and its a `const`, but it assumes `plugin.hot` might change between the `if` and the lambda call.

`!` is a syntax to indicate TS that we have intrinsic knowledge that in this position `plugin.hot` will not be null nor undefined and removes the compiler error. (The knowledge here being that `plugin.hot` will not be modified by `module.hot.accept()`).